inventree_kicad/serializers.py

```
@@ -285,15 +285,30 @@
             }
 
-        # Check if a parameter for setting visisble fields exists
-        ki_visible_fields_param = part.parameters.filter(
-            template__name=self.plugin.get_setting('KICAD_FIELD_VISIBILITY_PARAMETER', 'Kicad_Visible_Fields')
-        ).first()
-        ki_visible_fields = []
 
-        if ki_visible_fields_param:
-            ki_visible_fields = ki_visible_fields_param.data.split(',')
-            ki_visible_fields = [field.strip().lower() for field in ki_visible_fields]
+        # Find the value parameter value associated with this part instance
```

Flake8 (7.2.0)

[error] too many blank lines (2)

(E303)